pkgs/development/tools/phase/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib, stdenv, fetchurl, unzip, installShellFiles, makeWrapper, autoPatchelfHook, zlib, glibc}:
+
+let
+  pname = "phase";
+  version = "1.18.6"; # This will be auto-updated by update.sh
+  hashes = builtins.fromJSON (builtins.readFile ./hashes.json);
+
+  # Binary target map
+  systemInfo = {
+    "x86_64-linux" = {
+      archivePath = "Linux-binary";
+      archString = "linux_amd64";
+    };
+    "aarch64-linux" = {
+      archivePath = "Linux-binary-arm64";
+      archString = "linux_arm64";
+    };
+    "x86_64-darwin" = {
+      archivePath = "Darwin-binary";
+      archString = "macos_amd64";
+    };
+    "aarch64-darwin" = {
+      archivePath = "Darwin-binary-arm64";
+      archString = "macos_arm64";
+    };
+  };
+
+  system = stdenv.hostPlatform.system;
+  info = systemInfo.${system} or (throw "Unsupported system: ${system}");
+
+  runtimeDeps = [
+    stdenv.cc.cc.lib
+    zlib
+    glibc
+  ];
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/phasehq/cli/releases/download/v${version}/phase_cli_${info.archString}_${version}.zip";
+    hash = hashes.${system};
+  };
+
+  sourceRoot = ".";
+  
+  nativeBuildInputs = [
+    unzip
+    installShellFiles
+    makeWrapper
+    autoPatchelfHook  # This will automatically fix library references
+  ];
+
+  buildInputs = runtimeDeps;
+
+  unpackPhase = ''
+    runHook preUnpack
+    unzip $src
+    runHook postUnpack
+  '';
```

Comment:
```suggestion
```
shouldn't be needed given `unzip` in `nativeBuildInputs`

Reply:
removed.